### ISSUE TITLE
The stream is not playable if used with the CF template.

### ISSUE THREAD

**Issue by muratugureminoglu** [ant-media/Ant-Media-Server#5859]
### Short description

If a cluster structure is created using the CloudFormation template, Edge will not be able to play some of the instances.

### Environment

* Operating system and version: Ubuntu 20.04
* Java version: 11
* Ant Media Server version: 2.7.0
* Browser name and version: Chrome

### Steps to reproduce
1. Set up a cluster with a minimum of 2 edges using the CloudFormation template
2. Publish on Origin
3. Play on Edge

### Expected behavior
Edge should play the streams on the origin

### Actual behavior
Edge sometimes cannot play the streams
 
### Logs
[ant-media-server origin.log](https://github.com/ant-media/Ant-Media-Server/files/13492047/ant-media-server.origin.log)

[ant-media-server-edge.log](https://github.com/ant-media/Ant-Media-Server/files/13492048/ant-media-server-edge.log)

#### Origin Logs:
```
2023-11-28 16:49:31,053 [vert.x-eventloop-thread-4] WARN  i.a.e.c.webrtc.OriginServerEndpoint - JWT Token is not valid to access the stream in origin for streamId:stream1 codec:H264 height:480 appName:WebRTCAppEE
2023-11-28 16:49:31,053 [vert.x-eventloop-thread-4] WARN  i.a.e.c.webrtc.OriginServerEndpoint - Origin server cannot find the matchfor streamId:stream1 codec:H264 height:480 appName:WebRTCAppEE
2023-11-28 16:49:31,054 [vert.x-eventloop-thread-4] ERROR io.antmedia.filter.AbstractFilter - com.auth0.jwt.exceptions.SignatureVerificationException: The Token's Signature resulted invalid when verified using the Algorithm: HmacSHA256
	at com.auth0.jwt.algorithms.HMACAlgorithm.verify(HMACAlgorithm.java:55)
	at com.auth0.jwt.JWTVerifier.verify(JWTVerifier.java:299)
	at com.auth0.jwt.JWTVerifier.verify(JWTVerifier.java:283)
	at io.antmedia.filter.JWTFilter.isJWTTokenValid(JWTFilter.java:95)
	at io.antmedia.enterprise.cluster.webrtc.OriginServerEndpoint.lambda$start$0(OriginServerEndpoint.java:74)
	at io.antmedia.enterprise.cluster.webrtc.ClusterPacketHandler.handle(ClusterPacketHandler.java:77)
	at io.antmedia.enterprise.cluster.webrtc.ClusterPacketHandler.handle(ClusterPacketHandler.java:12)
	at io.vertx.core.net.impl.NetSocketImpl.lambda$new$1(NetSocketImpl.java:98)
```

#### Edge Logs:
```
2023-11-28 16:49:43,019 [vert.x-eventloop-thread-2] INFO  i.a.enterprise.webrtc.WebRTCAdaptor - get number of viewers for stream id stream1 is 1
2023-11-28 16:49:43,081 [vert.x-eventloop-thread-2] INFO  i.a.e.cluster.webrtc.EdgeClient - Edge client-480p for stream stream1 video socket closed with intention: false. Reason may be idle timeout(2secs)
2023-11-28 16:49:43,083 [vert.x-eventloop-thread-2] INFO  i.a.e.cluster.webrtc.EdgeClient - Edge client-480p for stream stream1 connect video on port 5000 for node group:default
2023-11-28 16:49:43,083 [vert.x-eventloop-thread-2] INFO  i.a.e.cluster.webrtc.EdgeClient - Edge client-480 for stream stream1 audio socket closed with intention: false.Reason may be idle timeout(2secs)
2023-11-28 16:49:43,085 [vert.x-eventloop-thread-2] INFO  i.a.e.cluster.webrtc.EdgeClient - Edge client-480 for stream stream1 connect audio on port 5000 to host:10.0.1.91 for node group: default
2023-11-28 16:49:43,086 [vert.x-eventloop-thread-2] INFO  i.a.e.cluster.webrtc.EdgeClient - Edge client-480p for stream stream1 is connected video on port 5000 to host:10.0.1.91 for node group:default
2023-11-28 16:49:43,086 [vert.x-eventloop-thread-2] INFO  i.a.e.cluster.webrtc.EdgeClient - It's already initialized for stream:stream1
2023-11-28 16:49:43,087 [vert.x-eventloop-thread-2] INFO  i.a.e.cluster.webrtc.EdgeClient - Edge client-480 for stream stream1 is connected audio on port 5000 to host: 10.0.1.91
```

**Comment by muratugureminoglu** [comment]
With the attached template, I could be reproduced only in 1 out of 10 attempts. 

[antmedia-aws-autoscale-template.txt](https://github.com/ant-media/Ant-Media-Server/files/13858873/antmedia-aws-autoscale-template.txt)

**Comment by mekya** [comment]
Hi,
I've encountered this problem while working on releasing 2.10.1. The problem is about the update time of the settings and timing issues. Instead of directly resolving the problem(because it seems map synch and it's hard to locate the bug), I've simplified the logic there so that there is no map. It updates the database if the update times of settings are different.

I've sent this refactor and I'm checking the CI if it works as expected  

FYI @burak-58 @muratugureminoglu
